saleor/graphql/order/mutations/order_confirm.py

```diff
@@ -73,11 +79,19 @@ def perform_mutation(cls, root, info: ResolveInfo, /, **data):
         payment = order_info.payment
         manager = get_plugin_manager_promise(info.context).get()
         app = get_app_promise(info.context).get()
+        webhook_events_for_map = WEBHOOK_EVENTS_FOR_ORDER_CONFIRMED
```

Comment:
Maybe just `webhook_events` instead of `webhook_events_for_map`? I struggled with the meaning of this variable.